gradle/versions.gradle

```diff
@@ -78,6 +78,8 @@ ext {
     openTelemetryVersion = "1.18.0"
     jetpackDataStoreVersion = "1.0.0"
     lifecycleKtxVersion="2.5.1"
+    AndroidCredentialsVersion="1.2.0"
```

Comment:
would these always be the same version?

Reply:
I had separate defined versions in case they ended up being different, but seems they intend to keep the versions the same, as of the stable release. Will use AndroidCredentialsVersion for both dependencies: 8c9f0190b49ae2fe1d2a3907c016bbce6186ce73